src/codeflare_sdk/ray/cluster/test_config.py

```diff
@@ -35,7 +35,17 @@ def test_default_cluster_creation(mocker):
     mocker.patch("kubernetes.client.ApisApi.get_api_versions")
     mocker.patch("kubernetes.client.CustomObjectsApi.list_namespaced_custom_object")
 
-    cluster = Cluster(ClusterConfiguration(name="default-cluster", namespace="ns"))
+    cluster = Cluster(
+        ClusterConfiguration(
+            name="default-cluster",
+            namespace="ns",
+            annotations={
+                "app.kubernetes.io/managed-by": "test-prefix",
+                "key1": "value1",
+                "key2": "value2",
+            },
+        )
+    )
```

Comment:
```suggestion
    cluster = Cluster(
        ClusterConfiguration(
            name="default-cluster",
            namespace="ns",
            annotations={
                "app.kubernetes.io/managed-by": "test-prefix",
                "key1": "value1",
                "key2": "value2",
            },
        )
    )
```
This should not be changed as the test is supposed to create a cluster with the 2 mandatory params (name and namespace)